### ISSUE TITLE
Registrars' phone, email and web address to automatic emails

### ISSUE THREAD

**Issue by vohmar** [internetee/registry#180]
There is a reference to the registrar holding the registration to a domain in automatic email notices sent by the registry ([https://github.com/internetee/registry/tree/master/app/views/mailers](https://github.com/internetee/registry/tree/master/app/views/mailers)), but no registrar contact data. The  e-mail messages should have registrar's **phone number**, **email address** and **web address** as well (if exists in registry db).

**Comment by artur-beljajev** [comment]
Extended registrar's info has been added to the following mailers:
- expiration
- force_delete"

**Comment by artur-beljajev** [comment]
I was unable to run the following mailers:
pending_update_rejected_notification_for_new_registrant
pending_update_expired_notification_for_new_registrant
pending_update_notification_for_new_registrant

So as soon as I get some clarification, what is the correct way to run them, I will add that info there as well (as a separate ticket, probably)

**Comment by artur-beljajev** [comment]
Extracted to #204

**Comment by vohmar** [comment]
registrar data missing (email are sent out):
* pending_deleted
* pending_update_request_for_old_registrant
* pending_update_notification_for_new_registrant
* pending_update_rejected_notification_for_new_registrant
* pending_update_expired_notification_for_new_registrant

OK
* force_delete

Error
* expiration (#186)

**Comment by artur-beljajev** [comment]
@vohmar Is this the correct subject for force_delete mailer - "Kustutusmenetluse teade"?

**Comment by artur-beljajev** [comment]
* Removed invalid logo url from force_delete mailer

**Comment by vohmar** [comment]
@artur-beljajev there is currently no notable issue with mailer subjects, lets focus on adding the missing content.

**Comment by artur-beljajev** [comment]
P.S. Please do not remove branch registry-186, since I am implementing #180 there as well.

**Comment by artur-beljajev** [comment]
in branch registry-186

**Comment by artur-beljajev** [comment]
Blocked by #180

**Comment by artur-beljajev** [comment]
P.S. the reason why I was unable to run them before was that I have tested it against staging branch, which was outdated.

**Comment by vohmar** [comment]
@artur-beljajev where is the branch that I can merge to test this? Email templates are currently missing from staging.

**Comment by artur-beljajev** [comment]
See branch above.

**Comment by artur-beljajev** [comment]
Fixed

**Comment by vohmar** [comment]
Solved in #186 with outstanding issues

**Comment by vohmar** [comment]
#186 is in production, closed
